api/v1beta1/spifilecontentrequest_types.go

```diff
@@ -0,0 +1,70 @@
+//
+// Copyright (c) 2021 Red Hat, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package v1beta1
+
+import metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+type SPIFileContentRequestSpec struct {
+	FilePath string `json:"filePath"`
+	RepoUrl  string `json:"repoUrl"`
+	Ref      string `json:"ref,omitempty"`
```

Comment:
What is this ref? Is it the commit hash? While that name works, it is not immediately obvious what this is. Would you please add some code comments here?